### ISSUE TITLE
Implement GitHub Action to Auto Deploy

### ISSUE THREAD

**Issue by juancwu** [LaurierCS/Website#130]
We should setup a pipeline to auto deploy the website. 

The `main` branch is the most up to date branch development wise, but we should not deploy when merging to `main` because we might introduce bugs that has not been properly tested before.

We should create a new branch called `live` and has the live version of the website. Once something is ready to be deployed, a PR is created to merge into `live`.
